src/applications/vaos/services/organization/transformers.js

```diff
@@ -31,12 +31,14 @@ export function transformParentFacilities(parentFacilities) {
       ],
       active: true,
       name: facility.authoritativeName,
-      address: {
-        line: [],
-        city: facility.city,
-        state: facility.stateAbbrev,
-        postalCode: null,
-      },
+      address: [
```

Comment:
Do we use this address anywhere else? I thought we did on the facility page, but looks like we don't actually.

Reply:
I couldn't find anywhere else we are using the Organization/system address besides the CC preferences page.